lib/modules/apostrophe-seo-global/index.js

```diff
@@ -1,4 +1,21 @@
 module.exports = {
   improve: 'apostrophe-global',
-  seo: false
+  seo: false,
+  construct: (self, options) => {
+    const seoGoogleFields = options.seoGoogleFields ? [
+      {
+        name: 'seoGoogleTrackingId',
+        label: 'Google Tracking ID',
+        type: 'string',
+        help: 'Tracking ID provided by Google for Google Analytics.'
```

Comment:
I think this should specify it's using the Tag Manager ID. I think the old code version would be different.

Reply:
They are using "Tracking ID" in docs, am I missing something?

https://support.google.com/analytics/answer/1008080?hl=en

Reply:
If that's consistent with their language then that's cool. I got my head in 2014 GA still.